additional_stats.py

```diff
@@ -0,0 +1,58 @@
+def calc_cost(row):
+    # print(row.columns)
+    if type(row['AllocTRES']) == float:
+        return 0
+    else:
```

Comment:
you can remove the `else` because you already `return` in line 4.
By remove the 'else' you reduce the cognitive complexity of the function